src/commands/containers/attachShellContainer.ts

```diff
@@ -20,7 +21,14 @@ export async function attachShellContainer(context: IActionContext, node?: Conta
         });
     }
 
-    let osType = await getDockerOSType(context);
+    let osType: DockerOSType;
+    try {
+        osType = await getDockerOSType(context);
```

Comment:
Get OS type from container instead of system? That would allow for Windows containers in ACI to work.

* Need the SDK to provide platform